moodle-scalable-cluster-ubuntu/azuredeploy.json

```diff
@@ -30,58 +30,96 @@
             },
             "type": "bool"
         },
+        "redisDeploySwitch": {
+            "defaultValue": true,
+            "metadata": {
+                "description": "Switch to deploy a redis cache or not"
+            },
+            "type": "bool"
+        },
         "vnetGwDeploySwitch": {
             "defaultValue": false,
             "metadata": {
                 "description": "Switch to deploy a virtual network gateway or not"
             },
             "type": "bool"
         },
+        "installObjectFsSwitch": {
+            "defaultValue": false,
+            "metadata": {
+                "description": "Switch to install Moodle Object FS plugins (with Azure Blob storage)"
+            },
+            "type": "bool"
+        },
         "installO365pluginsSwitch": {
             "defaultValue": false,
             "metadata": {
                 "description": "Switch to install Moodle Office 365 plugins"
             },
             "type": "bool"
         },
+        "installGdprPluginsSwitch": {
+            "defaultValue": false,
+            "metadata": {
+                "description": "Switch to install Moodle GDPR plugins. Note these require Moodle versions 3.4.2+ or 3.3.5+ and these will be included by default in Moodle 3.5"
+            },
+            "type": "bool"
+        },
         "installElasticSearchSwitch": {
             "defaultValue": false,
             "metadata": {
                 "description": "Switch to install Moodle ElasticSearch plugins & VMs"
             },
             "type": "bool"
         },
-        "storageAccountType": {
-            "defaultValue": "Standard_LRS",
-            "allowedValues": [
-                "Standard_LRS",
-                "Standard_GRS",
-                "Standard_ZRS"
-            ],
+        "htmlLocalCopySwitch": {
+            "defaultValue": true,
             "metadata": {
-                "description": "Storage Account type"
+                "description": "Switch to create a local copy of /moodle/html or not"
+            },
+            "type": "bool"
+        },
+        "siteURL": {
+            "defaultValue": "www.example.org",
```

Comment:
is this valid for all deployments?

Reply:
Yes. The install scripts take either an empty value or this default value to mean "configure it for me". In this situation the scripts generate a DNS name based on the LB name and create a self-signed certificate. This behavior is documented (not that I expect you to read the docs in a review, just letting you know it is documemented).

Reply:
sounds good